pages/common/k9s.md

```diff
@@ -0,0 +1,20 @@
+# k9s
+
+> CLI to view and manage Kubernetes clusters.
```

Comment:
```suggestion
> View and manage Kubernetes clusters.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

Reply:
Thanks for the suggestion!